website/templates/project/project_base.mako

```diff
@@ -14,6 +14,11 @@
 
 <%include file="project_header.mako"/>
 
+% if status and node:
+    ## WARNING: Need new heading alert messages? They must be added at both here and base.mako
+    <%include file="../alert.mako"/>
```

Comment:
Ah ok, thanks for the clarification. I didn't see the other conditional in base.mako. One more question: is `node` always guaranteed to be in the mako context? If so, how?